es5.md

```diff
@@ -177,7 +177,9 @@ An expression statement, i.e., a statement consisting of a single expression.
 ## Directive
 
 ```js
-interface Directive <: ExpressionStatement {
+interface Directive <: Node {
+    type: "ExpressionStatement";
+    expression: Expression;
```

Comment:
I think it's reasonable to have `Literal` here.

Reply:
Agreed